content/articles/understanding-support-vector-machine-algorithm/index.md

```diff
@@ -0,0 +1,165 @@
+- Whenever the number of observations is more than the sample size, this method works well.
+- It also saves memory because just a small portion of the determination function's training points are used (called the support vector).
+
+**Disadvantages**
+- Because of the time required to train while working with large data sets, performance suffers.
+- Data sets with higher noise, such as those in which target classes overlap, do not perform well.
+- Probability estimates in SVM are generated via a time-consuming five-fold cross-validation technique. SVC, the scikit-learn library's associated SVC approach, uses this technique.
+
+### How to select vector machines kernels
+On a completely random set of data, we usually don't know which core is perhaps the most efficient. Because we don't know anything about the data, we must start with the smallest hypothesis space and work our way up. In conclusion, a linear kernel would produce decent results if our data set is linearly divided; else, it will fail.
+
+Let's pretend your dataset just has two dimensions for ease of display. This is a resounding success. The RBF core's SVM is shown below.
+
+![Rbf kernel svm](/engineering-education/understanding-support-vector-machine-algorithm/smv.jpg)
+
+This data set appears to be handled equally well by the linear SVM and the RBF core SVM. How can we figure out the linear hypothesis to use? Occam's Razor comes to mind here. Because they require bigger training sets, metric models like linear support vector machines (SVMs) are more challenging to employ than SVMs with RBF kernels. The RBF kernel SVM is more expensive to train, but it is also more expensive to store the kernel matrix and project it into a high-dimensional `infinite` space in which the data is transmitted linearly. The developed model will be more costly to the many hyperparameters we need to adjust. Complex models may also be retrained more easily.
+
+Anyway, we should not be too concerned about the present polynomial kernel. Due to efficiency difficulties, this isn't as handy as you may imagine for (both computational and forecasting). Instead of using the Radial Basis Function kernel, use a linear SVM (or logistic regression) for basic problems. This is the general principle to keep in mind.
+
+The RBF kernel SVM's decision area is also linear. As a consequence, the RBF core SVM elevates the samples into a high-dimensional feature set with linear decision boundaries that may be used to differentiate classes.
+
+### Conclution
+ As evidenced in this tutorial, we have looked more into the Support Vector Machine machine learning algorithm. We discussed the model's working concept, how to construct it in Python, how to optimize its parameters, and its benefits and disadvantages.
+ 
+Here is the full code [repo link](https://gist.github.com/mblondel/586753/f740949d0336484567dd422fe53445ac8821f5b2) of Support Vector Machines.
```

Comment:
Was this the code that you used in the tutorial?

It's a totally different code